### PR TITLE
Lmaoo197

### PR DESCRIPTION
**Pull request by LewisWelch94**

### CODE REVIEW

**Comment by OnikNoor97** [comment]
No more check points commits. You’re over using them

File: Script/server.js [hunk 1]
```diff
@@ -67,5 +67,20 @@ function loadReporterFromServer(ticketId)
             headers: {'Content-Type': 'multipart/form-data' }
         })
     
+        return response;
+}
+function getActiveUsers()
```
Comment (OnikNoor97) [inline Script/server.js:72]: Remember. Server methods must have from server. 
GetActiveUsersFromServer()

Comment (LewisWelch94) [reply Script/server.js]: Done

File: Script/server.js [hunk 2]
```diff
@@ -69,7 +69,8 @@ function loadReporterFromServer(ticketId)
     
         return response;
 }
-function getActiveUsers()
+
+function GetActiveUsersFromServer()
```
Comment (OnikNoor97) [inline Script/server.js:73]: Fixing, methods should not begin with a capital letter

Comment (LewisWelch94) [reply Script/server.js]: fixed

**Pull request merged**